smart-doc/articles/assembly.asm.xml

```diff
@@ -99,6 +99,7 @@
         -->
       <!-- enter the platform identifier or a list of
         identifiers, separated by ; -->
+      <!-- For a full list, see https://confluence.suse.com/x/aQDWNg -->
```

Comment:
From the file structure above and below this comment it is not obvious a 'full list of *what*' is to be expected in Confluence.  Could you specify this in the comment? For example: 

'For a full list of meta tags and their possible values, see  ...'

Reply:
You are right. I've applied that in commit 475ed91. Thank you! :+1: